plugin/lightline.vim

```diff
@@ -22,5 +22,6 @@ augroup lightline
   autocmd CursorMoved,BufUnload * call lightline#update_once()
 augroup END
 
+command! -nargs=1 -complete=customlist,lightline#cs_complete LightlineTheme let g:lightline.colorscheme = <q-args> | call lightline#cs_apply()
```

Comment:
This is a command that allows you to change/try a colorscheme, with wildmenu completion.